src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/utils/Notes.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.utils;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Getter
+@EqualsAndHashCode
+@ToString
+public class Notes implements Serializable {
+
+    private static final long serialVersionUID = 1928058324454924191L;
+
+    private String noteType;
+    private String user;
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
+    private OffsetDateTime created;
+
+    private Notes(){
+
+    }
+
+    @JsonCreator
+    public Notes(@JsonProperty String noteType, @JsonProperty String user, @JsonProperty OffsetDateTime created) {
```

Comment:
We use the ParameterNamesModule in the JacksonConfiguration.class so I believe there's no need to specify the JsonProperty here.
See: https://github.com/FasterXML/jackson-modules-java8/tree/master/parameter-names

Reply:
you are right.